### DISCUSSION TITLE
Why is it so hard to go through the docs and find what I need ?

### DISCUSSION THREAD

**Discussion by younes-io** [ttscoff/doing#164] [Q&A]
As a new user, I find it very hard to find what I need to do, which is simply just deleting a task I added by mistake.
The wiki does not seem to be easy to go through, compared to what I'm used to in other CLIs.
For example, why is it so hard (impossible) to find how to delete a task I added by mistake ? (oh, and I don't want to archive the task or edit some file or whatever, I just want to type a simple doing command and get the task removed!
I even cloned the wiki and created a custom GPT, and it didn't find a way.. it suggested archiving and editing a task... 
Anyway, as a new user, the wiki doesn't seem to be intuitive at all.. and it's a bit disappointing because _doing_ is really a great tool!

**Comment by ttscoff** [comment]
Use `doing help` to find commands like `doing undo.`

**Comment by ttscoff** [comment]
To be fair, doing has so many subcommands that a typical CLI-style 
documentation would be even less readable than the wiki. That's why the 
wiki exists. I'm not sure how you're searching, but 'undo last command' 
should have given you the results you needed.

**Comment by younes-io** [comment]
thank you, @ttscoff 
I don't mind wiki or docs, as long as it's intuitive / easy to adapt with.. (maybe it's too subjective).. but for example, in my use case, "undo" doesn't work.. because I don't want to "Ctrl + Z" my last actions.. because my last actions do not represent the tasks I added by mistake.. that's why I'm looking for a "delete" or a "remove" subcommand...


EDIT: btw, is there a cheat sheet for doing commands ? (I know _doing help_ exists.. but a cheat sheet could be easier to grasp)

**Comment by ttscoff** [comment]
Try `doing select`, locate the entry you want to remove, then select 
`remove`.

-Brett

**Reply by younes-io** [reply]
alright, thank you
